PatientMonitoringSystem/PatientMonitoringSystem/Controllers/ModuleRowController.cs

```diff
@@ -49,10 +49,25 @@ public void UpdateMaxValue(int value)
 			moduleRowView.UpdateMaxValue(value);
 		}
 
+		public bool CheckValueBreached(int value) {
+			if (value > module.MaxValue)
+			{
+				Console.WriteLine("(Module: {0})BREACH of MAX value {1} with new value {2}", module.Name, module.MaxValue, value);
+				return true;
+			}
+			else if (value < module.MinValue)
+			{
+				Console.WriteLine("(Module: {0})BREACH of MIN value {1} with new value {2}", module.Name, module.MinValue, value);
+				return true;
+			}
+
+			return false;
+		}
+
 		public void UpdateCurrentReading()
 		{
 			var reading = module.GetCurrentReading();
-
+			bool breached = CheckValueBreached(reading);
```

Comment:
@harrytpstudent Doesn't look like this variable is being used yet.

Reply:
This was a piece of code that I had on another branch.  It passed the breached bool to the view which would then turn the integer scroller background color to red.  I just copied it across to see the modules generating values in the console output